tests/pytests/test_json.py

```diff
@@ -572,3 +572,84 @@ def testDifferentType(env):
     conn.execute_command('JSON.SET', 'doc:2', '$', r'{"t":"hello world"}')
     env.expect('FT.SEARCH', 'hidx', '*', 'NOCONTENT').equal([1L, 'doc:1'])
     env.expect('FT.SEARCH', 'jidx', '*', 'NOCONTENT').equal([1L, 'doc:2'])
+
+def test_NoSortableArray(env):
```

Comment:
@ashtul why you added this test? I thought this is the confusing behavior that we don't want no?

Reply:
@MeirShpilraien you can see that `value` == `None`.
The test shows that we ignore the SORTABLE flag.

Reply:
We remove supprt for tag on json for now